strapi/strapi
# Auth: Trim whitespace from fields

### What does it do?

- Trims whitespace from registration form fields (except password fields)
- Adds tests for registration form

### Why is it needed?

It fixes a bug and eases the registration process. 

### How to test it?

1. Delete your `.tmp` folder and reload admin app
2. Enter a firstname which contains whitespace around
3. Verify you can register

### Related issue(s)/PR(s)

- Fixes https://github.com/strapi/strapi/issues/15456
